examples/text-generation/run_generation.py

```diff
@@ -544,7 +546,7 @@ def generate():
         raw_dataset = raw_dataset.remove_columns([name for name in raw_dataset.column_names if name != column_name])
 
         # Set the prompt length to args.max_input_tokens if > 0 else default to 16
```

Comment:
Can you also update this comment for the -1 case?

Reply:
thanks, fixed